build.gradle

```diff
@@ -18,8 +18,8 @@ plugins {
     id 'java-library'
     id 'application'
     id 'com.palantir.docker' version '0.25.0'
-    id 'org.jetbrains.kotlin.jvm' version '1.6.21'
-    id "org.owasp.dependencycheck" version "8.1.2"
+    id 'org.jetbrains.kotlin.jvm' version '1.8.22'
+    id "org.owasp.dependencycheck" version "8.4.0"
```

Comment:
Please, add licenses check and git property plugins. You can look in codec-sailfish for example to find the requested plugins IDs